### ISSUE TITLE
Robust Video Matting with GPU doesn't work

### ISSUE THREAD

**Issue by Vbrawl** [royshil/obs-backgroundremoval#267]
When selecting the following, the mask won't render and freezes.
Inference device: GPU - DirectML
Segmentation Model: Robust Video Matting

**To Reproduce**

1. Go to Camera Filters
2. Click on Background Removal effect
3. Select as Inference device: GPU - DirectML
4. Select as Segmentation Model: Robust Video Matting
5. See that it doesn't render the mask (No mask updates)

**Expected behavior**
I would expect the mask to automatically update every X frames (as set in Calculate mask every X frame setting, default to 1)

**Screenshots**
Failed Attempt: ![BackgroundRemoverBug](https://user-images.githubusercontent.com/62301731/233339596-3f365a1b-6fd4-4ac7-aa90-84d5629cd765.png)

Successful Attempt with different Segmentation Model: ![BackgroundRemoverSuccess](https://user-images.githubusercontent.com/62301731/233339805-f7012725-c8a9-47d0-a81b-19609854beb6.png)

**Log and Crash Report**
[2023-04-20 13-21-29.txt](https://github.com/royshil/obs-backgroundremoval/files/11285133/2023-04-20.13-21-29.txt)

**Desktop (please complete the following information):**

 - OS: Windows 11
 - Browser: Firefox (Don't think it matters)
 - Plugin Version: 0.5.17
 - OBS Version: 29.0.2
 - GPU: NVIDIA GeForce GTX 1650

**Comment by umireon** [comment]
@royshil I found this error

```
\a\_work\1\s\onnxruntime\core\providers\dml\DmlExecutionProvider\src\MLOperatorAuthorImpl.cpp(2340)\onnxruntime.dll!00007FFD26EDE70A: (caller: 00007FFD26EDDC5B) Exception(3) tid(438c) 80070057 The parameter is incorrect.
```

**Comment by umireon** [comment]
Switching models during using DirectML EP causes some errors.

**Comment by umireon** [comment]
@royshil I want to check this issue again after v0.5.18 will be released.
V0.5.18 will include various fixes around DirectML and I'd like to check if these fixes fix this problem first.

**Comment by royshil** [comment]
@umireon for me these problems are resolved in 0.5.18 so i think we can close this issue

@Vbrawl do you want to verify the latest release fixed your problem?

**Comment by Vbrawl** [comment]
It works like a charm now, thanks :)
@umireon
@royshil
